src/main/resources/db/migration/V1_50__REMOVE_ARMS_PAROMS_CAPACITY.sql

```diff
@@ -0,0 +1,88 @@
+drop view app.team_capacity_breakdown_view;
```

Comment:
Not sure what changes are been made here... I will assume they are not controversial?!